examples/pendulum/pendulum.py

```diff
@@ -0,0 +1,110 @@
+import biorbd
+
+from biorbd_optim import OptimalControlProgram, OdeSolver
+from biorbd_optim.plot import PlotOcp
+from biorbd_optim.problem_type import ProblemType
+from biorbd_optim.objective_functions import ObjectiveFunction
+from biorbd_optim.constraints import Constraint
+from biorbd_optim.path_conditions import Bounds, QAndQDotBounds, InitialConditions
+
+
+def prepare_ocp(biorbd_model_path="eocar.bioMod"):
+    # --- Options --- #
+    # Model path
+    biorbd_model = biorbd.Model(biorbd_model_path)
+
+    # Problem parameters
+    number_shooting_points = 30
+    final_time = 2
```

Comment:
Please refer to the new way implemented in align_markers, this should be in the parameters of the function so the main below and the test can call this prepare_ocp with different values

Reply:
Actually, it is not implemented yet in align_marker, please refer to static_arm.py in muscle_driven_ocp for an example :)